ci/bamboo/build-manylinux-prototype.sh

```diff
@@ -1,101 +0,0 @@
-#!/bin/bash
```

Comment:
I would prefer it left in some subdir of, preferably where it was.  I have a hunch that, in practice, no one will use this outside of our own pipeline.  If someone else sees value in it, the fact that it's in `ci/` won't stop them from using it, nor would its presence elsewhere encourage anyone to use it any more than where it is.

i.e. I'd rather not break convention, here.